glean/src/core/pings/maker.ts

```diff
@@ -8,21 +8,25 @@ import DatetimeMetricType, { DatetimeMetric } from "../metrics/types/datetime.js
 import { Lifetime } from "../metrics/index.js";
 import TimeUnit from "../metrics/time_unit.js";
 import { ClientInfo, PingInfo, PingPayload } from "../pings/ping_payload.js";
-import PingType from "../pings/index.js";
-import Glean from "../glean.js";
+import CommonPingData from "../pings/ping_data_interface.js";
 import CoreEvents from "../events/index.js";
+import MetricsDatabase from "../metrics/database.js";
+import EventsDatabase from "../metrics/events_database.js";
+import { DebugOptions } from "../debug_options.js";
+import PingsDatabase from "./database.js";
```

Comment:
```suggestion
import MetricsDatabase from "../metrics/database.js";
import EventsDatabase from "../metrics/events_database.js";
import PingsDatabase from "./database.js";
import { DebugOptions } from "../debug_options.js";
```

uber nit: can we keep the databases together? 😄